tests/client/test_encryption_client.py

```diff
@@ -0,0 +1,37 @@
+import os
+import unittest
+
+from qingstor.sdk.client.encryption_client import EncryptionClient
+
+ZONE="test_zone"
+ENCRYPT_KEY=b"0"*32
+CONFIG="test_config"
```

Comment:
I think use `test_config` driectly is easy enough to understand so we don't need to use `CONFIG` for that.

The same for `BUCKET_NAME` , `ZONE`, `CLIENT`,  `IN_KWAGRS` ....